DOCSP-35176: Remove caveat about nested functions with GH deployment (#654)
mongodb/docs-app-services: source/functions.txt

Atlas Functions
===============

.. meta:: 

>   :description: Atlas Functions are executable, server-side JavaScript code that define your app's behavior.

Reviewer: 🤔 

I'm not sure I'm sold on "define your app's behavior." (I know this just came from the opening sentence.) What about...

```suggestion
  :description: Define Atlas Functions and call them manually or automatically to execute server-side JavaScript code.
```
